cmd/collection/main.go

```diff
@@ -203,11 +203,24 @@ func main() {
 			return err
 		}).
 		Module("sdk client connection options", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) error {
-			if len(accessNodeIDS) < common.DefaultAccessNodeIDSMinimum {
-				return fmt.Errorf("invalid flag --access-node-ids atleast %d IDs must be provided", common.DefaultAccessNodeIDSMinimum)
+			var anIDS []flow.Identifier
+
+			// if * provided use all ids in protocol state by default
+			if accessNodeIDS[0] == "*" {
+				anIDS, err = common.DefaultAccessNodeIDS(node.State.Sealed())
+				if err != nil {
+					return fmt.Errorf("failed to get default access node ids %w", err)
+				}
```

Comment:
I think we should explicitly disallow this configuration on Mainnet for now.
```suggestion
			if accessNodeIDS[0] == "*" {
				if node.ChainID == flow.Mainnet {
					return fmt.Errorf("must specify preferred access node IDs for non-test networks")
				}
				anIDS, err = common.DefaultAccessNodeIDS(node.State.Sealed())
				if err != nil {
					return fmt.Errorf("failed to get default access node ids %w", err)
				}
```